crates/sncast/src/state/state_file.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
We should use this new function in generate_transaction_entry_with_id probably

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1754/commits/4e0d449c0cf21cf524b71ba710d604e2108f2e6b